### PR TITLE
VIDEO-7602: iOS allow host to mute speakers

### PR DESCRIPTION
**Pull request by timrozum**
### JIRA link(s):

- [VIDEO-7602](https://issues.corp.twilio.com/browse/VIDEO-7602)

### Description

1. Added ability for the host to mute other speakers. Under the hood it sends a mute message from host to speaker via video room data track.

### Before review
* [ ] Added unit tests if necessary
* [x] Updated affected documentation
* [x] Verified locally with all effected platforms
* [x] Manually sanity tested running locally
* [x] Included screenshot as PR comment (if needed)
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Re-tested if necessary

https://user-images.githubusercontent.com/1930363/141373812-01eb0a51-915b-4104-936c-272a40955b41.mov

### CODE REVIEW

File: apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridView.swift [hunk 1]
```diff
@@ -45,8 +46,24 @@ struct SpeakerGridView: View {
                 GeometryReader { geometry in
                     LazyVGrid(columns: columns, spacing: spacing) {
                         ForEach($viewModel.onscreenSpeakers, id: \.self) { $speaker in
-                            SpeakerVideoView(speaker: $speaker)
-                                .frame(height: geometry.size.height / CGFloat(rowCount) - spacing)
+                            Group {
+                                if role == .host && !speaker.isMuted && !speaker.isYou  {
+                                    Menu(
```
Comment (aaalaniz) [inline apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridView.swift:51]: Does wrapping the `SpeakerVideoView` in `Menu` enable long clicking the video to mute?

**Review by aaalaniz — APPROVED** [review]

File: apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridView.swift (hunk 1 above)
Comment (timrozum) [reply apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridView.swift:51]: A single tap will bring up the "menu".

I could actually use a "context menu" instead which would be a long press and then it blurs everything on the screen except that view and the menu. But I think this is typically used when a single tap is already used for another operation like present a new screen. Or if it is an advanced feature that we want to make harder to get to.

The regular menu seemed like the way to go.

File: apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridView.swift [hunk 2]
```diff
@@ -45,8 +46,24 @@ struct SpeakerGridView: View {
...
+                                        content: {
+                                            Button(
+                                                action: { viewModel.muteSpeaker(speaker) },
+                                                label: { Label("Mute", systemImage: "mic.slash") }
```
Comment (piyushtank) [inline apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridView.swift:55]: Have you guys thought of the mute button disabling feature by admin?

Comment (timrozum) [reply apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridView.swift:55]: @piyushtank what specific functionality are you referring to?

The functionality in this PR allows a host to mute any other speaker very similar to the live audio app.

Comment (piyushtank) [reply apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridView.swift:55]: +1 Per our slack discussion

**Review by piyushtank — APPROVED** [review]

**Pull request merged**